src/AutoRest.CSharp/MgmtTest/Generation/MgmtBaseTestWriter.cs

```diff
@@ -309,6 +283,16 @@ public void WriteSchemaObjectExampleValue(CodeWriter writer, ObjectType sot, Exa
                 WriteSchemaObjectExampleProperties(writer, sot, ev, variableName, consumedProperties);
         }
 
+        protected void CreateTagWriterDelegate(string newVariableName, CSharpType valueType, ExampleValue ev)
```

Comment:
I was told that CodeWriterDelegate is the last resort in our generator, but I believe this is a proper usage here. I do not cannot think of a better solution given the case here.